.travis.yml

```diff
@@ -161,11 +172,29 @@ jobs:
       before_install:
         - ./scripts/install_prereqs.sh
       script:
+        # TODO investigate why macos tests hang for FirebaseAuth - keychain related?
```

Comment:
I guess so. The OS asks for password when you want to access to the keychain.

Reply:
I suspect that it's related to the password, but somehow the dynamic framework version works.